data/transition-sites/mod_sabre.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: mod_sabre
+whitehall_slug: ministry-of-defence
```

Comment:
Do/will the users who will be working on the mappings belong to MOD in Signon? If not we'll need to use `extra_organisation_slugs` to grant them access.